docusaurus/docs/running-tests.md

```diff
@@ -321,6 +321,7 @@ To help you make up your mind, here is a list of APIs that **need jsdom**:
 - [`ReactDOM.render()`](https://facebook.github.io/react/docs/top-level-api.html#reactdom.render)
 - [`TestUtils.renderIntoDocument()`](https://facebook.github.io/react/docs/test-utils.html#renderintodocument) ([a shortcut](https://github.com/facebook/react/blob/34761cf9a252964abfaab6faf74d473ad95d1f21/src/test/ReactTestUtils.js#L83-L91) for the above)
 - [`mount()`](https://airbnb.io/enzyme/docs/api/mount.html) in [Enzyme](https://airbnb.io/enzyme/index.html)
+- [`render`](https://testing-library.com/docs/react-testing-library/api/#render) in [Testing Library](https://testing-library.com/)
```

Comment:
Shall we also make this `render()` to be in line with the previously mentioned APIs? Otherwise, this looks good to me, nice catch :)